make.sh

```diff
@@ -188,13 +204,14 @@ echo "Remove script files and git/index"
 rm -rf make.sh
 rm -rf .github/workflows/test_install_script.yml
 rm -f .git/index
+rm -rf Scripts
 git reset
 
 if [[ -z "${CI}" ]]; then
     rm -rf fastlane/Tests
     rm -f set_up_test_firebase.sh
     rm -f set_up_test_testflight.sh
-    sh deliverable_setup.sh
+    sh set_up_deliverable.sh
```

Comment:
After completing this sh run, should we remove this `set_up_deliverable.sh`?
Reusing isn't valuable since we removed all stuff during setting up.

Reply:
`set_up_deliverable.sh` only opens the `Constants.swift` with Xcode. It is included so that user can modify `Constants.swift` without knowing where the file is located (deep within Fastlane folder).
The `make.sh` script prompt the user if they want to modify `Constants.swift` at project setup or later because some project might not have the necessary information for delivery yet.
We can keep this file for later use so we don't have to remove it.

Reply:
As seeing, `set_up_deliverable.sh` contains two actions: 1) choosing CI/CD platform, 2) opening `Constants.swift`. It isn't very clear if the first action does nothing, even accidentally removing files from the pipeline in case users choose a different one.

So I prefer to separate action modifying `constants` to another script file. What do you think?

Reply:
Sounds good. I will create an issue for that. For this PR let's keep it as is.